wagtaillinkchecker/management/commands/linkcheck.py

```diff
@@ -35,8 +35,9 @@ def handle(self, *args, **kwargs):
 
         print(f'Scanning {len(pages)} pages...')
         scan = broken_link_scan(site, run_sync, verbosity)
-        broken_links = ScanLink.objects.filter(scan=scan, crawled=True)
-        print(f'Found {len(broken_links)} broken links.')
+        total_links = ScanLink.objects.filter(scan=scan, crawled=True)
+        broken_links = ScanLink.objects.filter(scan=scan, broken=True)
+        print(f'Found {len(total_links)} total links, with {len(broken_links)} broken links.')
```

Comment:
Should the second one be `ScanLinks.objects.filter(scan=scan, crawled=True, broken=True)` or is it assumed if a link is broken it was crawled?